### PR TITLE
feat : added product listing page

### PR DESCRIPTION
**Pull request by raman1999**
image
image

### CODE REVIEW

File: screens/product listing page/product.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  margin-top: 1rem;
+}
+.bd-bottom {
+  border-bottom: 1px solid var(--theme-shade-3);
+}
+
+.flex-column {
+  justify-content: space-between;
+  padding: 1rem;
+  gap: 0.5rem;
```
Comment (Priyanshi-Tiwari) [inline screens/product listing page/product.css:75]: Suggestion: you can add this in utils and use it wherever needed.
Also you can use pixels for values less than 1 rem.

File: screens/product listing page/product.css [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+}
+
+.btn {
+  width: 70%;
+}
+.btn-wishlist {
+  position: absolute;
+  top: 10px;
+  right: 9px;
+  color: rgb(182, 182, 182);
```
Comment (tanishq20) [inline screens/product listing page/product.css:85]: You can use var() for colors.

**Review by tanishq20 — APPROVED** [review]
Everything looks good. Suggested very small change.

File: screens/product listing page/product.css (hunk 2 above)
Comment (Sameony) [inline screens/product listing page/product.css:85]: To maintain consistency, either use named variables or direct name colors. As it increases readability too

**Review by Sameony — APPROVED** [review]
Very descriptive code, great work.
Do look up and use more utility classes as it greatly reduces redundancy and makes css cleaner.

File: screens/product listing page/product.css (hunk 1 above)
Comment (raman1999) [reply screens/product listing page/product.css:75]: I have added other majorly used flex styling already in my utils, it's specifically used for here only

Comment (raman1999) [reply screens/product listing page/product.css:75]: Yes ,I will do it

File: screens/product listing page/product.css (hunk 2 above)
Comment (raman1999) [reply screens/product listing page/product.css:85]: Yes ,I forget to see that

**Pull request merged**